colossalai/shardformer/policies/llama.py

```diff
@@ -60,6 +60,11 @@ def module_policy(self) -> Dict[Union[str, nn.Module], ModulePolicyDescription]:
         else:
             norm_cls = RMSNorm
 
+        if self.pipeline_stage_manager:
+            use_zbv = self.pipeline_stage_manager.use_zbv
+        else:
+            use_zbv = False
```

Comment:
use_zbv = self.pipeline_stage_manager is not None and self.pipeline_stage_manager.use_zbv

Reply:
> use_zbv = self.pipeline_stage_manager is not None and self.pipeline_stage_manager.use_zbv

Fixed. More elegant now.